src/logic/operate.js

```diff
@@ -5,16 +5,16 @@ const operate = (numberOne, numberTwo, operation) => {
   const numTwo = Big(numberTwo);
   switch (operation) {
     case '+':
-      return numOne.plus(numTwo);
+      return numOne.plus(numTwo).toString();
     case '-':
-      return numOne.minus(numTwo);
+      return numOne.minus(numTwo).toString();
     case '÷':
       if (numTwo === 0) return 0;
-      return numOne.div(numTwo);
+      return numOne.div(numTwo).toString();
     case 'X':
-      return numOne.times(numTwo);
+      return numOne.times(numTwo).toString();
     case '%':
-      return numOne.mod(numTwo);
+      return numOne.mod(numTwo).toString();
```

Comment:
for safety check the second number is not `'0'` here too.

Reply:
Done! 👍